helix-term/src/ui/mod.rs

```diff
@@ -177,7 +179,12 @@ pub fn file_picker(root: PathBuf, config: &helix_view::editor::Config) -> FilePi
         files,
         move |path: &PathBuf| {
             // format_fn
-            path.strip_prefix(&root).unwrap_or(path).to_string_lossy()
+            Spans::from(
+                path.strip_prefix(&root)
+                    .unwrap_or(path)
+                    .to_string_lossy()
+                    .into_owned(),
```

Comment:
Ditto here, is into_owned really necessary

Reply:
How to do it without into_owned?

Reply:
Does using Cow::from() work?

Reply:
I added `From<Cow<str>>` impls for Span and Spans